Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
```

Comment:
OK thanks for the precisions I will make the changes during the day.   
Thank you.